etc/scripts/json2csv.py

```diff
@@ -114,9 +114,6 @@ def collect_keys(mapping, key_group):
             if not path.endswith('/'):
                 path = path + '/'
 
-        # alway create a root directory
-        path = '/code' + path
```

Comment:
There should still be a CLI option to prefix the path if a user wishes to do so.